### PR TITLE
Update the corresponding translation according to mdn/content#23519

### PR DESCRIPTION
**Pull request by DreamOfIce**
### Description

Sync the change in English version

### Additional details

https://github.com/mdn/content/pull/23519

### CODE REVIEW

**Review by hochan222 — APPROVED** [review]
Thanks LGTM for ko :)

**Review by yin1999 — APPROVED** [review]
LGTM for zh-CN. Thanks!

**Comment by yin1999** [comment]
@mfuji09 would you mind to have a look on this :)

File: files/ja/web/media/formats/video_codecs/index.md [hunk 1]
```diff
@@ -998,7 +998,7 @@ HEVCは独占的な形式で、多くの特許で保護されています。ラ
           </tbody>
         </table>
         <p>
-          Internet Explorer と Edge は、ハードウェアコーデックを搭載した機器でのみ HEVC に対応しています。
+          Internet Explorer、Chrome、Edgeは、ハードウェアコーデックを搭載した機器でのみHEVCをサポートします。
```
Comment (mfuji09) [inline files/ja/web/media/formats/video_codecs/index.md:1001]: Please insert between a Latin alphabet and Japanese letter. And サポート in this sentence is ambiguous.
```suggestion
          Internet Explorer、Chrome、Edge は、ハードウェアコーデックを搭載した機器でのみ HEVC に対応しています。
```

**Review by mfuji09 — CHANGES_REQUESTED** [review]
Please look at the correction in 'ja'.

**Comment by yin1999** [comment]
Let's merge this :)

**Pull request merged**